cluster.yml

```diff
@@ -13,6 +13,19 @@
   vars:
     ansible_connection: local
 
+- hosts: localhost
+  tasks:
+    - name: deploy warning for non kubeadm
+      debug:
+        msg: "DEPRECATION: non-kubeadm deployment is deprecated from v2.9. Will be removed in next release."
+      when: not kubeadm_enabled and not skip_kubeadm_warning
+
+    - name: deploy cluster for non kubeadm
+      pause:
+        prompt: "Are you sure you want to deploy cluster using the deprecated non-kubeadm mode. If you want to do that, Please press the 'Enter'!!!"
```

Comment:
Please remove `If you want to do that, Please press the 'Enter'!!!`

Reply:
done